site/src/examples/link/OpenInANewTab.tsx

```diff
@@ -0,0 +1,37 @@
+import { ReactElement } from "react";
+import { StackLayout, Link } from "@salt-ds/core";
+import { GithubIcon } from "@salt-ds/icons";
+
+export const OpenInANewTab = (): ReactElement => (
+  <StackLayout>
+ <p>
+    Using default icon:
```

Comment:
We should decide if we think having them in sentences like this is what we want versus in sections with theses as subtitles